packages/xarc-app-dev/src/config/opt2/xarc-options.ts

```diff
@@ -76,4 +76,9 @@ export type XarcOptions = {
    * options related to babel transpiler
    */
   babelOptions?: BabelOptions;
+
+  /**
+   * option for app cwd to be used from xarc-options
```

Comment:
what does "to be used from xarc-options" mean?

Reply:
Fixed - I just changed the comment for it